e2e/plugin-lighthouse-e2e/tests/collect.e2e.test.ts

```diff
@@ -30,7 +30,7 @@ describe('collect report with lighthouse-plugin NPM package', () => {
 
     const { code, stdout } = await executeProcess({
       command: 'npx',
-      args: ['@code-pushup/cli', 'collect', '--no-progress'],
+      args: ['@code-pushup/cli', 'collect', '--no-progress', '--verbose'],
```

Comment:
Is this a left over?

Reply:
Yes, sorry for not providing the context for this change. I missed that the e2e test started failing in the previous PR that hid audits with perfect scores behind the `verbose` flag. 

Without `verbose`, this test was passing as is for Ubuntu and macOS, but for Windows, it was failing as the output there would be "All 4 audits have perfect scores". I'm not sure where this discrepancy is coming from.

Reply:
Hmm... We should at least document this behaviour. 
Also I guess we should put a comment there IMO.

Reply:
Added a comment explaining why the test must be run in `verbose` mode.